src/main/resources/view/SoConnectTheme.css

```diff
@@ -255,6 +255,11 @@
     -fx-vgap: 10;
 }
 
+#todoTags {
+    -fx-hgap: 10;
+    -fx-vgap: 10;
+}
+
```

Comment:
Sorry, but I think it's better if this group is moved to line 288, after the `.cell_small_label_todo` haha. That way the todo styling is grouped together.

Reply:
Okay done!